src/layout/layout_entity.cpp

```
@@ -145,6 +148,13 @@ namespace moth_ui {
             trackJson.push_back(*track);
         }
         j["tracks"] = trackJson;
+        if (!m_discreteTracks.empty()) {
+            nlohmann::json discreteJson;
```

Clang (14.0.6)

[error] variable 'discreteJson' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)